### PR TITLE
Enable support for Maven Local

### PR DESCRIPTION
**Pull request by RyanHir**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=permwrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.0-bin.zip
```
Comment (democat3457) [inline gradle/wrapper/gradle-wrapper.properties:3]: what's the reason for switching gradle vesions?

File: build.gradle [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
+    group = 'com.github.frc5431'
 }
 
 dependencies {
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
-    
     implementation wpi.deps.wpilib()
+    implementation wpi.deps.vendor.java()
+
+    // Remove Native Zips with GradleRIO 2022
```
Comment (democat3457) [inline build.gradle:27]: does this comment mean to remove the bottom two lines with the next gradlerio version?

File: gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (RyanHir) [reply gradle/wrapper/gradle-wrapper.properties:3]: I was originally testing with GradleRIO 2022 which used 7.x, and then went back to 2021 which used 6.0, I was just matching its version.

File: build.gradle (hunk 2 above)
Comment (RyanHir) [reply build.gradle:27]: No, you would remove the following
```gradle
nativeZip wpi.deps.wpilibJni(wpi.platforms.roborio)
nativeDesktopZip wpi.deps.wpilibJni(wpi.platforms.desktop)
nativeZip wpi.deps.vendor.jni(wpi.platforms.roborio)
nativeDesktopZip wpi.deps.vendor.jni(wpi.platforms.desktop)
```

**Pull request merged**